eloquent/release-bionic-arm64-build.yaml

```diff
@@ -3,7 +3,7 @@
 ---
 abi_incompatibility_assumed: true
 jenkins_binary_job_label: buildagent_arm64 || eloquent_binarydeb_ubv8
-jenkins_binary_job_priority: 79
+jenkins_binary_job_priority: 78
```

Comment:
This jsut doesn't match what we do for Dashing. I would suggest to either update Dashing accordingly or keep Eloquent the same as Dashing.

Reply:
I will make the change. If the support level for armhf changes for eloquent we will have to remember to address the priority level here.